src/main/java/org/opensearch/jdbc/types/StructType.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.jdbc.types;
+
+import java.sql.Struct;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.LinkedHashMap;
+import java.util.Collections;
+
+
+
+import org.opensearch.jdbc.StructImpl;
+ 
+public class StructType implements TypeHelper<Struct> {
+
+    public static final StructType INSTANCE = new StructType();
+
+    private StructType() {
+
+    }
+
+    @Override
+    public String getTypeName() {
+        return "Struct";
+    }
+
+    @Override
+    public Struct fromValue(Object value, Map<String, Object> conversionParams) {
+        if (value == null) {
+          return null;
+        }
+        Map<String, Object> structKeyValues = (Map<String, Object>) value;
```

Comment:
is it possible this cast will fail?

Reply:
My experience showed that for structs it should not.

Reply:
I'd consider adding a `instanceof` check in the `if` above.